### PR TITLE
Add styles and message for invalid preferences #168173940

### PR DESCRIPTION
**Pull request by software-project**
https://www.pivotaltracker.com/story/show/168173940

### CODE REVIEW

File: app/javascript/components/lease_ups/LeaseUpApplicationsTable.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ const LeaseUpApplicationsTable = ({ listingId, dataSet, onLeaseUpStatusChange, o
     const trClassName = classNames(
       'rt-tr-status',
       statusClassName,
-      { 'is-invalid': rowInfo && isInvalid(rowInfo.original) }
+      { 'is-invalid is-disqualified': rowInfo && isInvalid(rowInfo.original) }
```
Comment (akegan) [inline app/javascript/components/lease_ups/LeaseUpApplicationsTable.js]: we might be able to drop the is-invalid piece? or do we need to update pattern lib to not make it greyed out?

Comment (software-project) [reply app/javascript/components/lease_ups/LeaseUpApplicationsTable.js]: We either need to update pattern lib or leave is-invalid in for grayed out text.

Comment (akegan) [reply app/javascript/components/lease_ups/LeaseUpApplicationsTable.js]: does is-invalid do anything other than grayed-out text? we actually want to get rid of the grayed out text piece

Comment (software-project) [reply app/javascript/components/lease_ups/LeaseUpApplicationsTable.js]: only that, will remove it then

**Review by akegan — DISMISSED** [review]
@software-project looks like removing the is-invalid class also got rid of the "NRHP Invalid" piece with the roll-over behavior

image

Seems like we should maybe update the pattern lib to have is-invalid just change the background color and not gray out the letters?

**Comment by software-project** [comment]
@akegan I suppose we need it then and styles update. Added it back.

**Comment by akegan** [comment]
changes look good with pattern lib update, but I want to confirm the final style with @emilyshore before we merge.

**Comment by akegan** [comment]
@callaghanc latest master of pattern lib is imported so this is ready for review!

**Review by callaghanc — APPROVED** [review]

**Pull request merged**